pkg/signatures/factory_test.go

```diff
@@ -25,20 +24,22 @@ MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEWi3tSxvBH7S/WUmv408nKPxNSJx6
 -----END PUBLIC KEY-----
 `
 	// b64Signature is a cosign signature b64 encoded.
-	b64Signature = "MEUCIDGMmJyxVKGPxvPk/QlRzMSGzcI8pYCy+MB7RTTpegzTAiEArssqWntVN8oJOMV0Aey0zhsNqRmEVQAYZNkn8h" +
-		"kAnXI="
+	b64Signature = "MEQCIDsFckfIg/uxqSGvf4UC4c9MzAVhuHwzq3NnYovbobYfAiAw31/xz56hS9xRl/xhRV7+RqOl3hXYi7UKO+q7q+" +
+		"kOxQ=="
 	// b64SignaturePayload is the payload associated with the cosign signature, it references the imgString.
-	b64SignaturePayload = "eyJjcml0aWNhbCI6eyJpZGVudGl0eSI6eyJkb2NrZXItcmVmZXJlbmNlIjoidHRsLnNoL2Q4ZDM4OTJkLTQ" +
-		"4YmQtNDY3MS1hNTQ2LTJlNzBhOTAwYjcwMiJ9LCJpbWFnZSI6eyJkb2NrZXItbWFuaWZlc3QtZGlnZXN0Ijoic2hhMjU2OmVlODli" +
-		"MDA1MjhmZjRmMDJmMjQwNWU0ZWUyMjE3NDNlYmMzZjhlOGRkMGJmZDVjNGMyMGEyZmEyYWFhN2VkZTMifSwidHlwZSI6ImNvc2lnb" +
+	b64SignaturePayload = "eyJjcml0aWNhbCI6eyJpZGVudGl0eSI6eyJkb2NrZXItcmVmZXJlbmNlIjoidHRsLnNoLzg4Nzk1ZGQ0LTI" +
+		"3MGMtNGViNy1iM2Q0LTUwMjQxZDViYzA0YyJ9LCJpbWFnZSI6eyJkb2NrZXItbWFuaWZlc3QtZGlnZXN0Ijoic2hhMjU2OmYyZTk4" +
+		"YWQzN2U0OTcwZjQ4ZTg1OTQ2OTcyYWM0YWNiNTU3NGMzOWYyN2M2MjRlZmJkOWIxN2EzYTQwMmJmZTQifSwidHlwZSI6ImNvc2lnb" +
 		"iBjb250YWluZXIgaW1hZ2Ugc2lnbmF0dXJlIn0sIm9wdGlvbmFsIjpudWxsfQ=="
 	// imgString points to a temporary available docker image reference, which was used to create the b64signature.
-	imgString = "ttl.sh/d8d3892d-48bd-4671-a546-2e70a900b702@sha256:ee89b00528ff4f02f2405e4ee221743ebc3f8e8dd0" +
-		"bfd5c4c20a2fa2aaa7ede3"
+	imgString = "ttl.sh/88795dd4-270c-4eb7-b3d4-50241d5bc04c@sha256:f2e98ad37e4970f48e85946972ac4acb5574c39f27" +
+		"c624efbd9b17a3a402bfe4"
 )
 
+var testBundle = []byte(`{"SignedEntryTimestamp":"MEUCIDStNB5vlCyLnF8Xr1vcGQ78Puw2xf72uUcFNOxrAMmUAiEAjv+IyBm9hd7IiodkmRvWL41CCenT58IYRdmRL+5W1MA=","Payload":{"body":"eyJhcGlWZXJzaW9uIjoiMC4wLjEiLCJraW5kIjoiaGFzaGVkcmVrb3JkIiwic3BlYyI6eyJkYXRhIjp7Imhhc2giOnsiYWxnb3JpdGhtIjoic2hhMjU2IiwidmFsdWUiOiJiOTY5NjUwZDI4YjQxYjNhNGM5NjQ0ZDM2NThkNDU5YmQ1NDZhOTdmNTVmZTU0M2ZhMzBhZDkwNTcyNDk3Mzk2In19LCJzaWduYXR1cmUiOnsiY29udGVudCI6Ik1FUUNJRHNGY2tmSWcvdXhxU0d2ZjRVQzRjOU16QVZodUh3enEzTm5Zb3Zib2JZZkFpQXczMS94ejU2aFM5eFJsL3hoUlY3K1JxT2wzaFhZaTdVS08rcTdxK2tPeFE9PSIsInB1YmxpY0tleSI6eyJjb250ZW50IjoiTFMwdExTMUNSVWRKVGlCUVZVSk1TVU1nUzBWWkxTMHRMUzBLVFVacmQwVjNXVWhMYjFwSmVtb3dRMEZSV1VsTGIxcEplbW93UkVGUlkwUlJaMEZGZDNvMllUaHZlRUo1U25FNWN6aHJRM2gyYXpkU1UzUjVaMjFFVmdvd2RWaFlOWEZaU0dKT05YTjRXVGhzWW14b1RHczVkVTl5TVc1R1QyaE9RVXAxWVRrMWVrdzJSWGREU1RKM1JubHJVbmR4WjBZeFFreG5QVDBLTFMwdExTMUZUa1FnVUZWQ1RFbERJRXRGV1MwdExTMHRDZz09In19fX0=","integratedTime":1747398492,"logIndex":214049199,"logID":"c0d23d6ad406973f9559f3ba2d1ca01f84147d8ffc5b8445c224f98b9591801d"}}`)
```

Comment:
how about moving it to testdata and load with `embed`? or maybe decode base64 (we have base64 in base64 here :)

Reply:
I wanted to avoid confusion with the other test bundle, but I can move it to a json file as well.